js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -696,6 +812,13 @@
         }
       );
       this.updateGraphManagementView();
+
+      // TODO: Currently whenever a user modifies the current graph model, and any of the above calls:
+      // "addEdgeDefinition", "modifyEdgeDefinition", "deleteEdgeDefinition" errors out -> we are notifying
+      // the user (which is good), but we're also closing the modalView directly. So the user has no chance
+      // to directly adjust his configuration to fix the misconfiguration. This needs to be improved.
+      // We need to gather all API modification results from above, and are only allowed to close this modal
+      // if all of them succeeded.
```

Comment:
`Promise.all()`? I think this might be important enough to not keep pending for later?